h2/test_h2_headers.py

```diff
@@ -106,3 +107,91 @@ def test_random_header(self):
         self.start_tempesta()
         self.start_all_clients()
         self.wait_while_busy(curl)
+
+class CurlTest(tester.TempestaTest):
+
```

Comment:
No description is provided, why the test is required? It doesn't check a specific test case. Or it was intended to be  a base class?